src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -11,6 +11,7 @@ internal class RuleBuilder
     {
         private readonly FakeAsserter.Factory asserterFactory;
         private readonly FakeManager manager;
+        private bool ruleAdded;

```

Comment:
Consider adding a prefix such as "was" to make this sound more like a Boolean than an event. (And a corresponding change to `EnsureRuleAdded`?)